### PR TITLE
Update mocaccino to v6 and mocha to v9

### PR DESCRIPTION
**Pull request by albertyw**
Pulls in mocaccino v6 and mocha v9.  Mocha v9 is used for internal testing of mochify.

Depends on https://github.com/mantoni/mocaccino.js/pull/38 to update mocaccino versions.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -85,7 +85,7 @@
   "devDependencies": {
     "@studio/changes": "^2.0.1",
     "eslint": "^7.13.0",
-    "mocha": "^8.4.0",
+    "mocha": "^9.1.0",
```
Comment (m90) [inline package.json:88]: Not sure if this your intention in this PR, but this is the Mocha version used by the tests in this repo. Mochify itself will still be on version 8 as per the version of mocaccino that is used. Is that the idea here?

Comment (albertyw) [reply package.json:88]: I wanted to check if mochify.js would be compatible with mocha v9, as per https://github.com/mantoni/mocaccino.js/pull/38#discussion_r774395444.

Comment (albertyw) [reply package.json:88]: I added another commit to also update mocaccino (temporarily to my fork) to see if there would be any conflicts with mocha.

**Review by m90 — APPROVED** [review]
LGTM. Are you ok if we make this `9.0.0` @mantoni ?

**Review by mantoni — APPROVED** [review]
@m90 Go! ❤️

**Comment by m90** [comment]
This is released as `9.0.0` :nail_care:

**Pull request merged**